src/btree/bt_curprev.c

```diff
@@ -47,16 +48,17 @@ __cursor_skip_prev(WT_CURSOR_BTREE *cbt)
 	 * If the search stack does not point at the current item, fill it in
 	 * with a search.
 	 */
-	while ((current = cbt->ins) != PREV_INS(cbt, 0)) {
+	recno = WT_INSERT_RECNO(cbt->ins);
+	while (((current = cbt->ins) != PREV_INS(cbt, 0)) &&
+	    (current != NULL)) {
```

Comment:
My hope is that adding the `recno` assignment outside the loop will clear up the Coverity warning because we'll no longer be indirecting through `current` after its assignment. So, the test for `current != NULL` should no longer be necessary.

Reply:
Yes, the NULL check is not required, I missed to remove during the modifications. 
Fixed in d1d819e.